### DISCUSSION TITLE
License of dependencies

### DISCUSSION THREAD

**Discussion by emmanuelbernard** [commonhaus/foundation#19] [Q&A]
Any restriction on dependencies of Cf projects
https://github.com/commonhaus/foundation-draft/blob/main/governance/ip-policy.md

Can it uses a GPL dependency ? How about proprietary, fauxpersource ?

**Comment by ebullient** [comment]
That's a good question re: required dependencies. 

GPL is an OSI-approved license, as long as use of GPL dependency (and implication to consumers) is well-documented, that wouldn't be an issue from my perspective.

Proprietary / fauxpersource would be an issue, as that could hurt the ability of the project to be viable over the long term.

**Comment by Sanne** [comment]
I believe I understand what you mean by _fauxpersource_ but googled it to confirm.. FYI zero responses so I'm slightly in doubt :)

**Comment by emmanuelbernard** [comment]
fauxpensource is a term I coined to describe BSL style licenses that are not open source for n years and then fall into the open source licence. The reason is that for me it kill the open and open source ecosystem to use these licenses.

**Comment by ebullient** [comment]
Guidance in the bylaws + eligibility for project would be OSI-approved licenses.

I do have follow up "technical detail" questions (how to actually do/verify the thing). e.g. license scanning tool to detect this. I can't easily find "the right" one. (IBM had one, I think RH probably has one... out in the great wide world, I can find only one, and it's maintenance/health is dubious... no idea)

**Reply by maxandersen** [reply]
Eclipse and cncf foundation have some too.

**Reply by ebullient** [reply]
yep. I just need to get my hands on the ones that are maintained and work. ;) Have pointers?

**Reply by kenfinnigan** [reply]
One of the popular tools CNCF projects use is: https://github.com/fossas/fossa-cli

**Comment by kenfinnigan** [comment]
https://github.com/commonhaus/foundation-draft/blob/main/policies/ip-policy.md#license-selection-and-usage covers the licenses that are supported, and how to seek approval for others.

If that doesn't answer the question sufficiently, please open a PR with proposed changes
